.github/workflows/build-pr.yml

```diff
@@ -114,10 +115,19 @@ jobs:
       - name: Save logs
         if: ${{ always() }}
         run: |
-          kubectl logs -n gatekeeper-system -l control-plane=controller-manager --tail=-1 > logs-externaldata-controller-${{ matrix.KUBERNETES_VERSION }}-${{ matrix.GATEKEEPER_VERSION }}.json
```

Comment:
why did we delete these?

Reply:
because we don't restart GK when we restart Ratify, so we can just export the log once at the last step.